build.gradle

```diff
@@ -121,19 +129,19 @@ repositories {
 }
 
 
-project.tasks['sonarqube'].dependsOn test, functional
-check.dependsOn functional
+project.tasks['sonarqube'].dependsOn test, integration
+check.dependsOn integration
 checkstyleMain.shouldRunAfter(compileJava)
 test.shouldRunAfter(checkstyleTest)
-functional.shouldRunAfter(checkstyleIntegrationTest)
-functional.shouldRunAfter(test)
+integration.shouldRunAfter(checkstyleIntegrationTest)
+integration.shouldRunAfter(test)
```

Comment:
Can we check that `functional` is supposed to be removed here? @iguannago @satyachundur